gen/bq/asgt/dataservice/v1alpha1/sample_data.schema

```diff
@@ -158,6 +158,50 @@
     "name": "created_at",
     "type": "TIMESTAMP",
     "mode": "NULLABLE"
+   },
```

Comment:
These should just in firestore and be ignored by the bq schema generation.

Reply:
ah nice.. didn't notice